_search-plugins/sql/protocol.md

```diff
@@ -329,3 +329,74 @@ Amber|Duke|32
 Dale|Adams|33
 Hattie|Bond|36
 ```
+
+## JDBC Format
+
+### Description
+
+The plugin provides responses in JDBC format. The JDBC format is widely used because it provides schema information and more functionality such as pagination. Besides JDBC driver, various clients can benefit from the detailed and well formatted response.
+
+**NOTE:** `JDBC` is the only format supported by the `PPL` endpoint, but it is one of several formats supported by the `SQL` endpoint.
```

Comment:
The _ppl endpoint supports jdbc, raw, and csv formats. It does not support json format.